routes/analytics.js

```diff
@@ -153,6 +168,12 @@ async function advancedAnalytics(req) {
 	return getAdvancedReports({ evType, publisher, channels })
 }
 
+async function adminAnalytics(req) {
+	const { channels, earner } = req.query
+	if (!channels) throw new Error('please provide channels query param')
+	return analytics(req, channels.split('+'), earner)
```

Comment:
there are two schools of thought here

you either pass multiple channels by doing `?channelId=..&channelId=...` in which case express will give you an array for channel ID

or you handle it manually as you do here...

I'm fine with either - I like this one more, so this is good :+1: